components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/internal/service/impl/password/PasswordRecoveryManagerImpl.java

```diff
@@ -638,10 +752,10 @@ private void invalidateRecoveryInfoSendCode(String resendCode, String notificati
      * @param recoveryData Data to be stored as mata which are needed to evaluate the recovery data object
      * @throws IdentityRecoveryServerException Error storing recovery data
      */
-    private void addRecoveryDataObject(String secretKey, String recoveryData, User user)
+    private void addRecoveryDataObject(String secretKey, String recoveryFlowId, String recoveryData, User user)
```

Comment:
update the method comment

Reply:
Fixed with ab489ffd47d3bb04acd840bdff99979c579d20aa